src/shibuya.js

```diff
@@ -112,8 +112,8 @@ const main = async () => {
     console.log(`requireWeightAtMost: ${requireWeightAtMost}`);
 
     console.log('\n5. Execute the above an XCM from Shibuya to schedule a task on Turing ...');
-    const totalInstructionWeight = 6 * SHIBUYA_INSTRUCTION_WEIGHT;
-    const fungible = 6255948005536808;
+    const totalInstructionWeight = 6 * SHIBUYA_INSTRUCTION_WEIGHT; 
```

Comment:
This number, SHIBUYA_INSTRUCTION_WEIGHT, is defined by Shibuya. How much is it on Turing? Maybe you should ask Nikhil.

Reply:
I I think INSTRUCTION_WEIGHT might be defined on Turing, since these instructions are executed on Turing.